diff --git a/node_modules/react-native/Libraries/PermissionsAndroid/NativePermissionsAndroid.js b/node_modules/react-native/Libraries/PermissionsAndroid/NativePermissionsAndroid.js
index 76b4742..5a69a8a 100644
--- a/node_modules/react-native/Libraries/PermissionsAndroid/NativePermissionsAndroid.js
+++ b/node_modules/react-native/Libraries/PermissionsAndroid/NativePermissionsAndroid.js
@@ -17,6 +17,7 @@ export type PermissionType = string;
 /*
 export type PermissionStatus = 'granted' | 'denied' | 'never_ask_again';
 export type PermissionType =
+  | 'android.permission.POST_NOTIFICATIONS'
   | 'android.permission.READ_CALENDAR'
   | 'android.permission.WRITE_CALENDAR'
   | 'android.permission.CAMERA'
diff --git a/node_modules/react-native/Libraries/PermissionsAndroid/PermissionsAndroid.js b/node_modules/react-native/Libraries/PermissionsAndroid/PermissionsAndroid.js
index 227af12..1b42627 100644
--- a/node_modules/react-native/Libraries/PermissionsAndroid/PermissionsAndroid.js
+++ b/node_modules/react-native/Libraries/PermissionsAndroid/PermissionsAndroid.js
@@ -34,6 +34,7 @@ const PERMISSION_REQUEST_RESULT = Object.freeze({
 });
 
 const PERMISSIONS = Object.freeze({
+  POST_NOTIFICATIONS: 'android.permission.POST_NOTIFICATIONS',
   READ_CALENDAR: 'android.permission.READ_CALENDAR',
   WRITE_CALENDAR: 'android.permission.WRITE_CALENDAR',
   CAMERA: 'android.permission.CAMERA',
